### ISSUE TITLE
Advanced Search Selectors - 4.1.2 Name, Role, Value - None of the select2 dropdowns are interpreted as expanded or collapsed on focus or user input

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#10108]
As a user reliant on the screen reader to inform me of when a component has changed state, for instance when a dropdown menu has expanded or a slider has been switched on, I expect the screen reader to interpret the item as 'expanded' or 'collapsed' on focus and on input.

**Linked Work Item:**
#9386 

**Expected Result:**
Items which change state on user input, such as dropdowns, sliders and checkboxes, should communicate their changed state to the user via the screen reader. This can be done using the 'aria-expanded' attribute. 

**Actual Result:**
 The dropdowns should interpret as expanded or collapsed, but when the user focuses on or opens the dropdowns, nothing is communicated via the screen reader to inform the user of whether the menu is open or closed.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/93d2b5ca-ab73-4f0b-9098-ed4e0f3761ec)

**Comment by apeters** [comment]
@imogenroberts this should be fixed in the latest

**Comment by imogenroberts** [comment]
RETEST: The advanced search panels are all now interpreted as expanded and collapsed on focus and on user input.

**Evidence:**


https://github.com/archesproject/arches/assets/77161085/28a071c5-91b7-46fc-a241-4fea3b509b3a
